tests/unit/connector/test_http_input.py

```diff
@@ -57,7 +60,9 @@ def setup_method(self):
         super().setup_method()
         self.object.pipeline_index = 1
         self.object.setup()
+        self.app = self.object.http_server.server.config.app
         self.target = self.object.target
+        self.falconClient = testing.TestClient(self.app)
```

Comment:
Please call it simply client. So we have not refactor it in case of switching to another framework. Additionally is CamelCase naming not pythonic.